src/PortingAssistantExtensionClientShared/Utils/AwsUtils.cs

```diff
@@ -40,6 +42,7 @@ public AwsCredential(string AwsAccessKeyId, string AwsSecretKey, string SessionT
     public static class AwsUtils
     {
         private static readonly SharedCredentialsFile sharedProfile = new SharedCredentialsFile();
+        public static bool isDismissed = true;
```

Comment:
for a public static we probably should be a little more descriptive of what this property is. Maybe AwsUtils.IsCredsNotificationDismissed